WordPress/gutenberg
# Comment Pagination Previous: Add Border and Spacing Support

## What?

Add border and Spacing support to the `Comments Pagination Previous` block.

Part of https://github.com/WordPress/gutenberg/issues/43247

## Why?
`Comments Pagination Previous` block is missing border and Spacing support.

## How?

Adds the border and Spacing support in block.json.

## Testing Instructions

- Go to Global Styles Settings. ( Under Appearance > Editor > Styles > Edit styles > Blocks )
- Make sure that `Comments Pagination Previous` block's border and Spacing is Configurable via Global Styles.
- Edit Template/ Single Post Template &  Add `Comments Pagination Previous` block and Apply the border Styles and Spacing.
- Verify that `Comments Pagination Previous` block styles take precedence over global Styles.
- Verify that `Comments Pagination Previous` block borders and Spacing display correctly in both the Editor and Frontend.

## Screenshots or Screencast 


https://github.com/user-attachments/assets/dedf0a97-d0d4-405c-8dc9-bc07fffed7b1
